src/fstext/pre-determinize-inl.h

```diff
@@ -547,9 +547,9 @@ void PreDeterminize(MutableFst<Arc> *fst,
     for (typename map<pair<StateId, ArcId>, size_t>::iterator m_iter = m_map.begin();
         m_iter != m_map.end();
         ++m_iter) {
-      pair<StateId, ArcId> pr = m_iter->first;
-      StateId state = pr.first;
-      ArcId arcpos = pr.second;
+      pair<StateId, ArcId> m_pr = m_iter->first;
```

Comment:
Kirill, since this m_pr is only used in the following two lines, I think it would better just to remove the variable entirely.
And then you can leave the other variable with its old name.